### PR TITLE
Small documentation tweaks

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Just add a few TODO items and improve our guidelines.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [x] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/PROCESS.md [hunk 1]
```diff
@@ -73,13 +73,14 @@ For FerretDB Inc. engineers and long-term contributors, besides the guides in [C
    (because [words like "refs" do not link PR to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)),
    but don't forget to reopen the issue once PR is merged,
    but the issue as a whole is not done.
-5. Please create a draft pull request as soon as you start working on an issue; that is needed for our process tooling.
+5. (This point is for FerretDB Inc. engineers only.)
```
Comment (noisersup) [inline .github/PROCESS.md:76]: If I understand we mention that, as PROCESS.md applies also to long-term contributors

**Review by noisersup — APPROVED** [review]

File: .github/PROCESS.md (hunk 1 above)
Comment (AlekSi) [reply .github/PROCESS.md:76]: Yes, and also because we ask for the opposite in CONTRIBUTING.md

**Review by w84thesun — APPROVED** [review]
👍

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by Fashander — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Pull request merged**